doc/installing.rst

```diff
@@ -18,17 +23,18 @@ Compiling the grammar
 	This step is optional - Blackbird is packaged with the ANTLR grammar
 	already pre-compiled, for both C++ and Python.
 
-To compile the grammar, you will need to install ANTLR 4.7.1:
+To compile the grammar, you will need to make sure that Java version 1.6 and higher
+is installed. Once Java is installed, use the following steps to install ANTLR 4.7.1:
 
 .. code-block:: console
 
-    $ cd $HOME/.loca/.lib
+    $ cd $HOME/.local/.lib
```

Comment:
What if the user does not have this directory?

Reply:
Oh, I think I am losing my mind. Typo on top of a typo. It should be `/usr/local/lib`, and all performed with sudo